source/reference/options.txt

```diff
@@ -468,6 +474,52 @@ Authentication Options
    - ``none``, the effect is the same as setting
      ``authMechanismProperties=CANONICALIZE_HOST_NAME:false``.
 
+.. option:: --oidcFlows
+
+   Specifies OpenID Connect flows in a comma-separated list. 
+   The OpenID connect flows specify how ``mongosh``interacts with the identity 
+   provider for the authentication process. ``mongosh`` supports the following 
+   OpenID Connect flows:
+
+   .. list-table::
+      :header-rows: 1
+      :widths: 25 75
+
+      * - OpenID Connect Flow
+        - Description
+
+      * - ``auth-code``
+        - Default. ``mongosh`` redirects you to the identity provider log-in screen.
+
+      * - ``device-auth``
+        - ``mongosh`` provides you with a URL and code to finish authentication.
+          This is considered a less secure OpenID Connect flow.
+
+   To set ``device-auth`` as a fallback option to ``auth-code``, see the following 
+   example:
+
+   .. code-block:: bash 
+
+      mongosh 'mongodb://localhost/' --authenticationMechanism MONGODB-OIDC --oidcFlows=auth-code,device-auth
+
+.. option:: --oidcRedirectUri
+
+   Specifies a URI where the identity provider redirects you after authentication.
+   The URI must match the configuration of the identity provider used by the server.
```

Comment:
```suggestion
   The URI must match the configuration of the identity provider.
```

(Having deja vu here, I might have already commented this somewhere else, this is not related to the configuration of the MongoDB server)

Reply:
Ah, that was https://github.com/mongodb/docs-compass/pull/508/files#diff-e1fa5430d5ec6ced67a5a7b640244598b3cca80a7c07c48ac1222b4c0264a81eR109 – that comment is marked as resolved without changes but I still think it makes sense to update it both here and in the Compass docs

Reply:
Done.